includes/admin/customers/customers.php

```diff
@@ -800,7 +800,8 @@ function edd_customers_view( $customer = null ) {
 						<div class="add-customer-email-wrapper">
 							<input type="hidden" name="customer-id" value="<?php echo esc_attr( $customer->id ); ?>" />
 							<?php wp_nonce_field( 'edd-add-customer-email', 'add_email_nonce', false, true ); ?>
-							<input type="email" name="additional-email" value="" placeholder="<?php esc_html_e( 'Email Address', 'easy-digital-downloads' ); ?>" />&nbsp;
+							<label class="screen-reader-text" for="additional-email"><?php esc_html_e( 'Email Address', 'easy-digital-downloads' ); ?></label>
+							<input type="email" name="additional-email" id="additional-email" value="" placeholder="<?php esc_html_e( 'Email Address', 'easy-digital-downloads' ); ?>" />&nbsp;
```

Comment:
Can you prefix the ID with `edd-`? Or maybe `edd-customer-`. Leave the name the same though.

I know there are plenty of other places in the file that don't have this prefix, but really we should. So let's add it for any new IDs we're creating.